deploy/Dockerfile.all-in-one

```diff
@@ -3,9 +3,12 @@ USER 0
 WORKDIR /work
 COPY ./ .
 
-RUN go mod tidy
-RUN go build -o domainproxyserver cmd/server/main.go
-RUN go build -o domainproxyclient cmd/client/main.go
+ENV GOTOOLCHAIN=auto \
```

Comment:
What's the purpose of these two env variables?

Reply:
Ah, I see that `go mod tidy` failed in a previous run, I suppose that might necessitate the env variables depending on build environment... Never failed when building domain proxy in JBS. I wonder if the top level JBS go config avoided that.

Reply:
Yes its slightly bizarre that it didn't fail in JBS but those variables were used elsewhere in JBS so I suspect you're right and there was some subtle interaction that made it work.